### DISCUSSION TITLE
Error: Invariant: static generation store missing in revalidateTag jsonplace

### DISCUSSION THREAD

**Discussion by ghost** [vercel/next.js#54649] [App Router]
When I fetch API in the index page on button click,
code:-
initial
...
 {
next:{
tag:["mytag"]
}
}
...
and on click of button  revalidateTag("mytag"),
It gives me an error like:- 
`Error: Invariant: static generation store missing in revalidateTag jsonplace`

**Comment by IntuzParthD** [comment]
Facing same issue.
